Expensify/App
# Fix : android crash when attaching a large size image

### Details

 * fixe App crashes when attaching a large size image in android chat 
   
### Fixed Issues

$ https://github.com/Expensify/App/issues/6836

### Tests



### QA Steps

1- Open app and login
2- Open any chat
3- Tap on + > Add Attachment > Choose from galery
4- Select a very long image 
5- Wait a moment
6- Verify that app not crash 


### Tested On


### Screenshots


#### Web


#### Mobile Web


#### Desktop


#### iOS


https://user-images.githubusercontent.com/21364901/149563567-73a28f4b-0010-4407-a652-e00fd84cdd31.mov


#### Android


https://user-images.githubusercontent.com/21364901/149562930-8a6ecb97-094d-4638-b36e-ced086a98047.mov
